src/js/me-shim.js

```diff
@@ -835,11 +849,17 @@ mejs.YouTubeApi = {
 					pluginMediaElement.success(pluginMediaElement, pluginMediaElement.pluginElement);
 
 					mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'canplay');
-					
-					// create timer
-					setInterval(function() {
-						mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'timeupdate');
-					}, 250);
+
+					var originalSeekTo = settings.pluginMediaElement.pluginApi.seekTo.bind(player);
+
+					settings.pluginMediaElement.pluginApi.seekTo = function(seconds, allowSeekAhead) {
+						originalSeekTo(seconds, allowSeekAhead);
+						// Allow player to update before triggering timeupdate event
+						setTimeout(function() {
+							mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'seeked');
+							mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'timeupdate');
+						}, 100);
```

Comment:
i know we spoke offline about this magic number 100.  i wish we had a more robust solution, but i don't have a suggestion.  one other concern i didn't think of before -- could this cause a confusing sequence of events?  if there's an interval spewing `timeupdate` events every 250ms, it seems possible we'd actually do the `originalSeekTo`, the player would seek, the interval would fire a `timeupdate` for the new time, _then_ this timeout would fire the `seeked`.  so it looks like the time jumped before the seek.  confusing?  a bit.  problematic (e.g. for our analytics)?  i don't know.